galicaster/classui/recorderui.py

```diff
@@ -422,6 +421,13 @@ def update_scheduler_timeout(self, status, event_type, title):
                         status.set_name('blinking_coloured_to')
                     status_label_blink = not status_label_blink
 
+            elif self.recorder.title_standin is not None:
+                if event_type.get_text():
+                    event_type.set_text("")
+                if status.get_text():
+                    status.set_text("")
+                title.set_text(_(self.recorder.title_standin))
+
             else: # Not current or pending recordings
```

Comment:
@androidwiltron Thanks for the recommendations and the plugin to test the PR :)
 I wrote the if statement to be the same as the exisitng one and if the title_standin is not se (None) then it would default to the "No upcoming events" (statement below the change).